lighthouse-core/lib/network-recorder.js

```diff
@@ -107,9 +107,15 @@ class NetworkRecorder extends EventEmitter {
         return;
       }
 
+      // QUIC network requests don't always "finish" even when they're done loading data
+      // Use receivedHeaders instead, see https://github.com/GoogleChrome/lighthouse/issues/5254
+      const isQUIC = record._responseHeaders && record._responseHeaders
+          .find(header => header.name.toLowerCase() === 'alt-svc' && /quic/.test(header.value));
```

Comment:
how certain is the `'alt-svc'` check? Any other way to tell?

Reply:
seems to be fairly certain, in the spec it's not technically required to announce it buuuuuut...

> While HTTP/QUIC doesn't formally require a client to implement Alt-Svc, there's no discovery mechanism other than Alt-Svc provided, so you're not going to get very far without 
it. 

from https://github.com/quicwg/base-drafts/issues/253